docs/reference/ingest/processors/user-agent.asciidoc

```diff
@@ -21,6 +21,7 @@ The ingest-user-agent module ships by default with the regexes.yaml made availab
 | `target_field`         | no        | user_agent                                                                                      | The field that will be filled with the user agent details.
 | `regex_file`           | no        | -                                                                                               | The name of the file in the `config/ingest-user-agent` directory containing the regular expressions for parsing the user agent string. Both the directory and the file have to be created before starting Elasticsearch. If not specified, ingest-user-agent will use the regexes.yaml from uap-core it ships with (see below).
 | `properties`           | no        | [`name`, `major`, `minor`, `patch`, `build`, `os`, `os_name`, `os_major`, `os_minor`, `device`] | Controls what properties are added to `target_field`.
+| `extract_device_type`  | no        | `false`                                                                                         | (beta) Extracts device type from the user agent string on a best-effort basis
```

Comment:
I'd use the beta admon to get the auto-generated text on rollover.
<img width="411" alt="Screen Shot 2021-04-19 at 9 31 35 AM" src="https://user-images.githubusercontent.com/40268737/115244628-1a609880-a0f2-11eb-93d5-b6b8db38122b.PNG">

I'd also add a period for consistency.

```suggestion
| `extract_device_type`  | no        | `false`                                                                                         | beta:[] Extracts device type from the user agent string on a best-effort basis.
```